site/mkdocs.yml

```diff
@@ -106,6 +106,7 @@ nav:
     - Spec: spec.md
   - Project:
     - How To Release: how-to-release.md
+    - How To Vote: how-to-vote.md
```

Comment:
How about "How to verify a release"? It isn't about voting, it's about validation.

Reply:
Great point, fixed!